### PR TITLE
Adopt AEP-160: Filtering

### PR DESCRIPTION
**Pull request by rofrankel**
## 🍱 Types of changes

What types of changes does your code introduce to AEP? _Put an `x` in the boxes
that apply_

- [ ] Enhancement
- [ ] [New proposal](https://aep.dev/1#workflow)
- [x] Migrated from google.aip.dev (actually from Roblox's AIPs in this case)
- [ ] Chore / Quick Fix

## 📋 Your checklist for this pull request

Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for
contributing to this repository.

### General

- [x] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
- [x] Ensure that your PR
      [references AEPs](https://aep.dev/style-guide#referencing-aeps)
      correctly.
- [x] [My code has been formatted](https://aep.dev/contributing#formatting)
      (usually `prettier -w .`)

### Additional checklist for a new AEP

- [ ] A new AEP **should** be no more than two pages if printed out.
- [ ] Ensure that the PR is editable by maintainers.
- [ ] Ensure that [File structure](https://aep.dev/style-guide#file-structure)
      guidelines are met.
- [ ] Ensure that
      [Document structure](https://aep.dev/style-guide#document-structure)
      guidelines are met.

💝 Thank you!

### CODE REVIEW

File: aep/general/0160/aep.md.j2 [hunk 1]
```diff
@@ -1,5 +1,54 @@
 # Filtering
```
Comment (toumorokoshi) [inline aep/general/0160/aep.md.j2:1]: it looks like aep.yaml is still in review - switch to approved?

**Review by toumorokoshi — APPROVED** [review]
LGTM - just a quick nit.

File: aep/general/0160/aep.md.j2 [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+operators defined by CEL, but only permit a certain number of them (to avoid
+"queries of death" or other performance concerns).
+
+Further structure or limitations **must** be clearly documented, **must not**
+violate requirements set forth in this document, and a non-compliant filter
+query **must** error with `INVALID_ARGUMENT`.
+
+<!-- prettier-ignore-start -->
+[cel]: https://github.com/google/cel-spec
+[CEL language definition]: https://github.com/google/cel-spec/blob/master/doc/langdef.md
```
Comment (gibson042) [inline aep/general/0160/aep.md.j2:53]: Should this be a versioned reference?

Comment (rofrankel) [reply aep/general/0160/aep.md.j2:53]: Not sure, so filed https://github.com/aep-dev/aep.dev/issues/179 to track.

Comment (rofrankel) [reply aep/general/0160/aep.md.j2:53]: (resolving so I can submit, following the now-established practice of trying to make changes in aip.dev adoption PRs as non-blocking as possible)

**Pull request merged**